packs/travisci/actions/build_repo.yaml

```diff
@@ -0,0 +1,14 @@
+name: build_repo
```

Comment:
Can I pass an arbitrary git `ref` to this action? It's good to be able to kick off builds against `master`, but equally beneficial to be able to kick off builds against any branch.